.circleci/config.yml

```diff
@@ -627,6 +627,22 @@ jobs:
             - redis.results
       - *save_cache_step
 
+  rediscluster:
+    docker:
+      - *test_runner
+      - image: grokzen/redis-cluster:4.0.9
+        env:
+          - IP=0.0.0.0
+    steps:
+      - checkout
+      - *restore_cache_step
+      - run: tox -e '{py27,py34,py35,py36}-rediscluster{135}' --result-json /tmp/rediscluster.results
```

Comment:
After the latest refactorings, this should be `rediscluster_contrib-{py27,py34,py35,py36}-rediscluster{135}`